glean_parser/templates/markdown.jinja2

```diff
@@ -28,9 +28,27 @@ See the Glean SDK documentation for the [`{{ ping_name }}` ping]({{ ping_name|pi
 {% if ping_name|ping_send_if_empty %}
 This ping is sent if empty.
 
+{% endif %}
+{% if ping_name|ping_include_client_id %}
+This ping includes the client id.
+
+{% endif %}
+{% if ping_name|ping_data_reviews %}
+**Data reviews for this ping:**
+{% for review in ping_name|ping_data_reviews %}
+    - [{{review}}]({{review}})
```

Comment:
If `review` is an actual URL markdown has a short variant for that: `<url>`

Reply:
Hm, documentation says these shoud be URIs. So I guess there is no guarantee they can be linked.

From https://mozilla.github.io/glean_parser/pings-yaml.html:
> A list of URIs to any data collection reviews relevant to the ping.

Reply:
These _are_ URIs, so above is fine, we can use them as link.
I just offered a different way of expressing that.

Reply:
My point is that the documentation states URI, which doesn't necessarily mean URL, which means it might not be linkable at all.

Reply:
Probably overthinking this, hehe. I do prefer the <url> syntax and didn't know it before :) I'll change it.